import-service/src/api/import-products-file.ts

```diff
@@ -0,0 +1,48 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import { getLog } from './helpers/get-log';
+import { S3 } from 'aws-sdk';
+import { jsonResult } from './helpers/json-result';
+import { validate } from 'jsonschema';
+import { invalidInputError } from './errors/invalid-input.error';
+import { cannotImportFileError } from './errors/cannot-import-file.error';
+
+export const importProductsFile: APIGatewayProxyHandler = async (
+  event,
+  context
+) => {
+  const { log, logCall } = getLog(event, context);
+  logCall();
+
+  const validationResult = validate(event.queryStringParameters, {
```

Comment:
**suggestion**: If we try to request url without passing file name in querystrings params (https://r9tp5o4mh9.execute-api.eu-west-1.amazonaws.com/dev/import) we receive 502 error, maybe it because is absent `type: 'object'` in validation schema, what do you think?

Reply:
It's just because I was being lazy to configure API Gateway properly.